moscar.go

```diff
@@ -1,124 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"github.com/micromdm/squirrel/munki/datastore"
-	"github.com/micromdm/squirrel/munki/models"
-	"html/template"
-	"io"
-	"log"
-	"net/http"
-	"time"
-)
-
-const STATIC_URL string = "/static/"
-const STATIC_ROOT string = "static/"
-const REPO_PATH string = "/Users/Shared/repo"
-
-type Context struct {
-	Static string
-	Prods  []ParsedPkginfo
-}
-
-// Template expects icon, 'product' name, description, and version, which becomes the download URL
-type ParsedPkginfo struct {
-	Icon        string
-	Name        string
-	Descript    string
-	Version     string
-	DownloadURL string
-}
-
-func Parse(in_pkginfo *models.PkgsInfo, ppi *ParsedPkginfo) *ParsedPkginfo {
```

Comment:
`in_pkginfo` would be better named `in` or `input`, the type following tells you what it is already. 
Also see the comment about using `camelCase` instead of `snake_case`.